buildconfig/pygame-stubs/color.pyi

```diff
@@ -18,3 +18,4 @@ class Color(object):
     def normalize(self) -> Tuple[float, float, float, float]: ...
     def correct_gamma(self, gamma: float) -> Color: ...
     def set_length(self, len: int) -> None: ...
+    def lerp(self, color: Color, value: float) -> Color: ...
```

Comment:
Is a type in the range 0.0 to 1.0 possible?
The keywords are color, and amount according to color.c:861
`    static char *keywords[] = {"color", "amount", NULL};`

Reply:
I changed keyword, and for range I will try https://github.com/python/typing/issues/554